pandas/tests/io/json/test_pandas.py

```diff
@@ -1908,3 +1908,15 @@ def test_complex_data_tojson(self, data, expected):
         # GH41174
         result = data.to_json()
         assert result == expected
+
+    def test_json_uint64(self):
+        # GH21073
+        data = {"col1": [13342205958987758245, 12388075603347835679]}
+        expected = (
+            '{"columns":["col1"],"index":[0,1],'
+            '"data":[[13342205958987758245],[12388075603347835679]]}'
+        )
+        orient = "split"
```

Comment:
You don't have to define orient or data as variables, just put them into the functions directly

Reply:
Thanks for the feedback! This is my first contribution to open source, so bear with me :)